web/static/js/components/questionnaires/QuestionnaireTitle.jsx

```diff
@@ -34,14 +34,20 @@ class QuestionnaireTitle extends Component {
     return (
       <div className='title-container'>
         <EditableTitleLabel title={questionnaire.name} onSubmit={(value) => { this.handleSubmit(value) }} emptyText={t('Untitled questionnaire')} readOnly={readOnly} more={<QuestionnaireMenu />} />
-        <EditableDescriptionLabel description={questionnaire.description} emptyText={t('Add description')} onSubmit={(value) => { this.handleSubmitDescription(value) }} readOnly={readOnly} />
+        <EditableDescriptionLabel description={questionnaire.description} emptyText={readOnly ? '' : t('Add description')} onSubmit={(value) => { this.handleSubmitDescription(value) }} readOnly={readOnly} />
```

Comment:
If it is readonly it should not even have an `onSubmit` attribute, right?

Reply:
I tried, but `onSubmit` is required. Should we change that?

Reply:
ok, no worries.